layouts/community/ortho_4x12/buswerks/keymap.c

```diff
@@ -7,6 +7,7 @@ extern keymap_config_t keymap_config;
 #define _DVORAK 2
```

Comment:
You may want to consider using an enum for the layers here, so you don't have to manage then numbers here.